packages/design-system/src/components/Button/style.scss

```diff
@@ -131,9 +134,29 @@
 
   .btn__loader {
     position: absolute;
-    top: 50%;
-    transform: translate(-50%, -50%);
-    left: 50%;
+    top: 0;
+    left: 0;
+    opacity: 1;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    padding: inherit;
+  }
+
+  .btn__loader-spinner {
+    min-width: 15px;
+  }
+
+  .btn__loader-label {
+    text-align: left;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    color: inherit;
+    margin: 0 5px 1px;
```

Comment:
do not understand this `margin-bottom: 1px;`, the label is centered with flexbox. Is it not working on some browser?

Reply:
I've had a feeling that the label wasn't aligned on the y-axis, but it seems to be good now. I will push a fix in sec